android/hello_sdl_android/src/main/java/com/sdl/hellosdlandroid/SdlReceiver.java

```diff
@@ -35,4 +37,13 @@ public void onSdlEnabled(Context context, Intent intent) {
     public void onReceive(Context context, Intent intent) {
         super.onReceive(context, intent); // Required if overriding this method
     }
+
+    private boolean isVehicleTypeSupported(Intent intent) {
+//        if(intent.hasExtra(TransportConstants.START_ROUTER_SERVICE_SDL_ENABLED_CONNECTED_VEHICLE_INFO))
+//        {
+//            VehicleType vehicleType = intent.getParcelableExtra(TransportConstants.START_ROUTER_SERVICE_SDL_ENABLED_CONNECTED_VEHICLE_INFO);
+//            /* Return true if vehicle is supported, else false*/
+//        }
+        return false;
```

Comment:
Please return true from this method. Since this is a sample app, it should connect to all vehicles.